views/homework/extended.hbs

```diff
@@ -73,10 +73,20 @@
             {{/userHasPermission}}
         {{/inArray}}
         {{#if courseId._id}}
-            <a class="btn btn-secondary" href="/courses/{{courseId._id}}" alt="zum Kurs der Aufgabe">zum Kurs</a>
+            <a href="/files/courses/{{courseId._id}}" class="btn btn-add btn-secondary">

```

Comment:
do we need a permission check here?

Reply:
there is a check if a course exists
otherwise, everybody can access the course files, right?